src/XIVLauncher/Settings/EnvironmentSettings.cs

```diff
@@ -1,17 +1,25 @@
 ﻿using System;
 using System.Collections.Generic;
 using System.Linq;
+using System.Runtime.InteropServices;
 using System.Text;
 using System.Threading.Tasks;
 
 namespace XIVLauncher
 {
     internal static class EnvironmentSettings
     {
-        public static bool IsWine => CheckEnvBool("XL_WINEONLINUX");
+        public static bool IsWine => IsLinux || IsMac;
+        public static bool IsLinux => CheckEnvBool("XL_WINEONLINUX");
+        public static bool IsMac => GetProcAddress(GetModuleHandle("ntdll.dll"), "wine_get_version") != IntPtr.Zero;
```

Comment:
Rather than set the `IsMac` bool here like this, I think we'd benefit from a `XL_WINEONMAC` environment variable. Then we can adjust `IsWine` accordingly because it's also possible for Linux users to turn off Wine exports.

The current `IsMac` check would probably best be turned into something like `FoundWineExports` and we could always do something like this to handle all possible cases.
```
IsWine = IsLinux || IsMac || FoundWineExports;
```

This is mostly to handle any potential future cases where we need to make platform-specific changes. There are quite a few for Linux and we can't guarantee that Wine on Linux and CrossOver on macOS will always be perfectly in sync in terms of experience, features, or bugs.

Reply:
Should be added in with 42196e2aeeb8e638be2a7b9ba3ae0591ae3f9bde